src/share/export.js

```diff
@@ -0,0 +1,48 @@
+import fs from '../lib/fs';
+const sh = new fs.Shell();
+import WebTorrent from 'webtorrent';
+
+var client = new WebTorrent();
+const files = [];
+
+document.getElementById('btnSeed').addEventListener('click', startSeeding);
```

Comment:
Be careful here that we don't allow double/triple/etc clicking, and restart seeding.  You should deal with multiple attempts to click this when it's already in progress.